tds/src/main/java/thredds/server/wfs/WFSGetFeatureWriter.java

```diff
@@ -15,14 +15,50 @@
 	private PrintWriter response;
 	private String fileOutput;
 	
+	
 	/**
-	 * Initiate the response with an XML file with an XML header and the FeatureCollection tag.
+	 * Initiate the response with an XML file with an XML header and the FeatureCollection tag. Write bounding box information.
 	 */
 	public void startXML() {
 		fileOutput += "<?xml version=\"1.0\" encoding=\"UTF-8\"?>";
 		fileOutput += "<wfs:FeatureCollection xsi:schemaLocation=\"http://www.opengis.net/wfs/2.0 http://schemas.opengis.net/wfs/2.0/wfs.xsd\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\""
-				+ " xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:gml=\"http://opengis.net/gml\" xmlns:fes=\"http://www.opengis.net/fes/2.0\" xmlns:ogc=\"http://www.opengis.net/ogc\""
-				+ " xmlns:ows=\"http://www.opengis.net/ows/1.1\" xmlns:wfs=\"http://opengis.net/wfs/2.0\" xmlns=\"http://www.opengis.net/wfs/2.0\" version=\"2.0.0\">";
+				+ " xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:gml=\"http://opengis.net/gml/3.2\" xmlns:fes=\"http://www.opengis.net/fes/2.0\" xmlns:ogc=\"http://www.opengis.net/ogc\""
+				+ " xmlns:wfs=\"http://opengis.net/wfs/2.0\" xmlns:tds=\"http://localhost:8080/thredds/wfs/results/tds\" xmlns=\"http://www.opengis.net/wfs/2.0\" version=\"2.0.0\" numberMatched=\"1\" numberReturned=\"1\">"

```

Comment:
Thanks! Yeah, they show up a lot in standards-based web services because most standards are composed of other standards which are composed of other standards... at some point the recursion stops, but it can be deeeep.

Reply:
Yeah it's a deep cavern system that can be pretty gritty to navigate!